distro.go

```diff
@@ -171,11 +188,21 @@ func (d Distro) GetConfiguration() (c Configuration, e error) {
 
 // String deserializes a distro and its configuration as a yaml string.
 func (d Distro) String() string {
+	return fmt.Sprintf("name: %s\n%s\n%s", d.Name(), d.guidToString(), d.configToString())
+}
+
+func (d Distro) guidToString() string {
```

Comment:
I would documente that private method, as it’s not clear without reading the implementation what it does on error. Also, it’s only used one, in `String()`, so maybe those are the same thing? (One caller, one implementation which is few lines -> it means it should be in the same code block)

Reply:
ok, so only documentation on what this method does

Reply:
(or second option: merging that one in String(), and only keep configToString() separted to keep the early returns)

Reply:
Done